### PR TITLE
Add replication policy doc

### PR DESCRIPTION
**Pull request by chivalryq**
Signed-off-by: Qiaozp <qiaozhongpei.qzp@alibaba-inc.com>

### Description of your changes

Also fix v1.6 Chinese document tag error:
image

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [x] Update `sidebar.js` if adding a new page.
- [x] Run `yarn start` to ensure the changes has taken effect.

### CODE REVIEW

File: docs/end-user/policies/replication.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+But projects like [OpenYurt](https://openyurt.io) have finer-grained division like node pool. This requires to dispatch some similar resources to the same cluster. 
+These resources are called replication. Back to the example of OpenYurt, it can integrate KubeVela and replicate the resources then dispatch them to the different node pool.
+
+## Usage
+
+To replicate component, `replication` policy is added as a built-in policy. It can be only used together with `deploy` workflow step.
+If using `replication` policy in `deploy` workflow step, a new field `context.replicaKey` will be added to when rendering the component. You can use this field to dispatch the resources to the same cluster.
+
+:::note
+`replication` policy is only supported in KubeVela version 1.6.0 or later.
```
Comment (wonderflow) [inline docs/end-user/policies/replication.md]:
```suggestion
`replication` policy is only supported in KubeVela version 1.6.0+.
```

File: docs/end-user/policies/replication.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+## Example
+
+The following ComponentDefinition is an example which make use of `replication` policy.
+
+:::note
+ComponentDefinition below is trimmed for brevity. See complete YAML in [replica-webservice](https://github.com/kubevela/kubevela/blob/master/test/e2e-test/testdata/definition/replica-webservice.yaml)
+:::
+
+```yaml
```
Comment (wonderflow) [inline docs/end-user/policies/replication.md]: 用 CUE 的方式写

File: docs/end-user/policies/replication.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+        					}
+        				}
+        				ports: exposePorts
+        				type:  parameter.exposeType
+        			}
+        		}
+        	}
+        }
+```
+
```
Comment (wonderflow) [inline docs/end-user/policies/replication.md:159]: 加上 vela def apply 的命令，引导用户执行

File: docs/end-user/policies/replication.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+        type: deploy
+        properties:
+          policies: ["replication-default"]
+```
+
+Then application will dispatch two deployments and two services:
+
+
+```shell
+kubectl get deploy -n default
```
Comment (wonderflow) [inline docs/end-user/policies/replication.md]: 用 vela status --detail --tree 查看

**Review by wonderflow — COMMENTED** [review]
LGTM after comments, please add chinese doc

File: i18n/zh/docusaurus-plugin-content-docs/current/installation/standalone.mdx [hunk 5]
```diff
@@ -325,7 +325,7 @@ VelaUX 是 KubeVela 的仪表板。 它是在你的集群中运行的 Web 应用
 如果你不使用 KubeVela 的 UI 控制台，可以跳过。
 
 ```shell script
-vela addon enable ~/.vela/addons/velaux
+vela addon enable ~/.vela/addons/velaux serviceType=NodePort
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/installation/standalone.mdx:328]: why this is needed?

Comment (chivalryq) [reply i18n/zh/docusaurus-plugin-content-docs/current/installation/standalone.mdx:328]: Align with English version. This is missed at first time. 
PS: In this part, user already have a server with public IP.

**Review by wonderflow — APPROVED** [review]

File: i18n/zh/docusaurus-plugin-content-docs/current/end-user/policies/replication.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
+---
+title: 组件复制
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/end-user/policies/replication.md]: 组件分裂

**Pull request merged**